less/home-page.less

```diff
@@ -18,14 +20,36 @@
     display: flex;
     flex-wrap: wrap;
     width: 100%;
+    @media @phone{
+      .sky{
+        display:none;
+      }//closes sky
+
+    }//closes media
+    @media @phone{
+      .launch{
+        display:none;
```

Comment:
It looks like you're comfortable with the @phone variable for media queries and nesting 👍